hw/top_earlgrey/data/top_earlgrey.sv.tpl

```diff
@@ -614,6 +618,7 @@ else:
             % if sig["act"] == "requester":
       .${sig["name"]}_o(),
             % elif sig["act"] == "receiver":
+              ## TODO: Add logic struct default value
```

Comment:
I am just curious: Is the default there mainly in case the other end has not yet defined the connection?

Reply:
Correct.